index.md

```diff
@@ -1,51 +1,67 @@
 
-The key features are:
-- Streaming - all data is streamed
-- Batch support - a collection of health data records can be streamed and processed together
-- Multitenancy - supports segregation of data by tenant and Data Integrator
+## Key features
 
-## Key Technologies
-- [Event Streams](https://www.ibm.com/cloud/event-streams), an IBM Cloud-based [Apache Kafka](https://kafka.apache.org/) managed service, is the technology used for producing and consuming the data streams
-- Using [IBM Cloud Functions](https://cloud.ibm.com/functions/learn/concepts), HRI exposes a Serverless RESTful Management API that is used to control and configure the system
-- [ElasticSearch](https://github.com/elastic/elasticsearch) is the distributed NoSQL data store that is used to store information about batches
+- **Streaming:** All data is streamed.
+- **Batch support:** A collection of health data records can be streamed and processed together.
+- **Multi-tenancy:** Supports segregation of data by [tenant](glossary.md#tenant) and Data Integrator
 
-## IBM Cloud Dependencies  
-The HRI was developed on the IBM cloud and currently does not support running on other public or private clouds. However, as a part of Alvearie, the HRI is being transitioned into an open source project, and the goal is to support public and private cloud deployments. The latest version has been pushed to GitHub, and the team continues to work on moving all development activities into the open and making the HRI support other clouds. Please see the [Roadmap](roadmap.md) for additional details.   
+## Key technologies
+
+- [**Event Streams**](glossary.md#event-streams): This is an IBM Cloud-based [Apache Kafka](https://kafka.apache.org/) managed service, the technology used for producing and consuming the data streams.
+- [**IBM Cloud Functions**](glossary.md#ibm-cloud-functions): Using these, the Health Record Ingestion service exposes a serverless RESTful Management API that is used to control and configure the system. 
+- [**Elasticsearch**](glossary.md#elasticsearch): This is the distributed NoSQL data store that is used to store information about batches.
+
+## IBM Cloud dependencies
+
+The Health Record Ingestion service was developed on the IBM Cloud and currently does not support running on other public or private clouds. However, as part of Project Alvearie, the Health Record Ingestion service is being transitioned into an open source project, with a goal of supporting public and private cloud deployments. The latest version has been pushed to GitHub. The team continues to work on moving all development activities into the open, and making the Health Record Ingestion service support other clouds. For more information about future plans, see the [Roadmap](roadmap.md).   
 
 ## Core Architecture
+
+
 ![core-architecture](assets/img/architecture-core.png)
+**Figure: Components of the Health Record Ingestion service**
+
+
+
```

Comment:
The word 'Figure:' is getting dropped. Normally, you would add an extra line, but in this case I don't think we want that. I played around locally and you need to add a `\<br\>` after the image import if we don't want an extra line.  Also please remove the extra blank lines. One is sufficient.
```
![core-architecture](assets/img/architecture-core.png)<br>
**Figure: Components of the Health Record Ingestion service**
```